### PR TITLE
ENH: added cross reference node->elements

### PR DESCRIPTION
**Pull request by saullocastro**
- this commit will allow to get not only `bdf.elements[eid].nodes` but also `bdf.nodes[nid].elements`

### CODE REVIEW

**Comment by saullocastro** [comment]
@SteveDoyle2 I understand that the CELAS can be grounded, but that seems not to affect the `bdf.nodes[nid].elements` generated by this pull request. Could you please explain it better?

**Comment by SteveDoyle2** [comment]
For a CELAS2, the nodes can be either:

    [1, 2]  # both grids

or

    # one grid
    [1, None]
    [None, 2]

where the None indicates the node has been grounded.  Similarly, for a CTETRA10, any of the last 6 nodes can be None, which means there is no midside node.

The SPOINT comment was wrong.  I was thinking of an old version of the code.  I think it'll work if you just add an:

    if node is None:
        continue

**Comment by saullocastro** [comment]
@SteveDoyle2 Thank you for the feedback, please check the new commit

File: pyNastran/bdf/bdfInterface/crossReference.py [hunk 1]
```diff
@@ -275,6 +279,19 @@ def _cross_reference_elements(self):
...
 
+    def _cross_reference_nodes_with_elements(self):
+        """
+        Links the nodes to all connected elements
+        """
+        nodes = defaultdict(set)
+        for element in self.elements.values():
+            if element.nodes is not None:
+                for node in element.nodes:
+                    if node is None:
```
Comment (saullocastro) [inline pyNastran/bdf/bdfInterface/crossReference.py]: @SteveDoyle2 here it goes the `None` check...

**Pull request merged**